tests/config/test_types.py

```diff
@@ -218,6 +263,20 @@ def test_deserialize_respects_optional(self):
         assert cv.deserialize(b"") is None
         assert cv.deserialize(b" ") is None
 
+    def test_deserialize_utilises_transformer(self):
+        import codecs
```

Comment:
`codecs` is in the stdlib, so shouldn't be a problem to move to the top of the file.

Reply:
I've made this change :+1: